articles/azure-stack/azure-stack-update-1903.md

```diff
@@ -153,26 +160,25 @@ The following are post-installation known issues for this build version.
 
 - An Ubuntu 18.04 VM created with SSH authorization enabled will not allow you to use the SSH keys to sign in. As a workaround, use VM access for the Linux extension to implement SSH keys after provisioning, or use password-based authentication.
 
-- Azure Stack now supports Windows Azure Linux agents greater than version 2.2.20. This support was a part of 1901 and 1902 hotfix and allows customers to maintain consistent linux images between Azure and Azure Stack.
-
+- Azure Stack now supports Windows Azure Linux agents greater than version 2.2.20. This support was a part of 1901 and 1902 hotfix and allows customers to maintain consistent Linux images between Azure and Azure Stack.
 
 - If you do not have a Hardware Lifecycle Host (HLH): before build 1902, you had to set group policy **Computer Configuration\Windows Settings\Security Settings\Local Policies\Security Options** to **Send LM & NTLM – use NTLMv2 session security if negotiated**. Since build 1902, you must leave it as **Not Defined** or set it to **Send NTLMv2 response only** (which is the default value). Otherwise, you won't be able to establish a PowerShell remote session and you will see an **Access is denied** error:
 
-   ```shell
-   PS C:\Users\Administrator> $session = New-PSSession -ComputerName x.x.x.x -ConfigurationName PrivilegedEndpoint  -Credential $cred
-   New-PSSession : [x.x.x.x] Connecting to remote server x.x.x.x failed with the following error message : Access is denied. For more information, see the 
+   ```powershell
```

Comment:
This isn't correct -- the powershell code specifier only delineates PowerShell code -- not PowerShell cmdlet output. Please remove this edit.

Reply:
This is Powershell code and I submitted it under 1902 issues, I wrote the command, and I wrote this bug, and I gave you the solution so I would know ;-)

Reply:
Or to be more precise it is an output from Powershell command run with parts of the actual command :-) but still it is better served as `powershell` than `shell` block

Reply:
https://github.com/MicrosoftDocs/azure-docs/commit/087ba22d26982109f84315334e115ec9ca1e32cf#diff-75bbfee4f79554fbf2c6ca9ae35ee451

Reply:
Can you please change it back to ```shell? Thanks again!

Reply:
Check the link above, I can change it to `shell` sure but it will make it wrong ;-P

Reply:
It even renders wrong as it is not highlighting anything: 
![image](https://user-images.githubusercontent.com/28607803/55836288-0b63aa80-5b16-11e9-9de2-4ab5aaf9b88a.png)

Reply:
OK, we can go with ```powershell, that's fine.